src/topology/constructions.lean

```diff
@@ -930,13 +930,17 @@ quotient_map_quot_mk
 lemma continuous_quotient_mk : continuous (@quotient.mk α s) :=
 continuous_coinduced_rng
 
-lemma continuous_quotient_lift {f : α → β} (hs : ∀ a b, a ≈ b → f a = f b)
-  (h : continuous f) : continuous (quotient.lift f hs : quotient s → β) :=
+lemma continuous.quotient_lift {f : α → β} (h : continuous f) (hs : ∀ a b, a ≈ b → f a = f b) :
+  continuous (quotient.lift f hs : quotient s → β) :=
 continuous_coinduced_dom.2 h
 
-lemma continuous_quotient_lift_on' {f : α → β} (hs : ∀ a b, a ≈ b → f a = f b)
-  (h : continuous f) : continuous (λ x, quotient.lift_on' x f hs : quotient s → β) :=
-continuous_coinduced_dom.2 h
+lemma continuous.quotient_lift_on' {f : α → β} (h : continuous f) (hs : ∀ a b, a ≈ b → f a = f b) :
```

Comment:
```suggestion
lemma continuous.quotient_lift_on' {f : α → β} (h : continuous f) (hs : ∀ a b, s.r a b → f a = f b) :
```
is the correct type here:

https://github.com/leanprover-community/mathlib/blob/bae872c3b7234f3aef80755d0f2ece656a7a43cd/src/data/quot.lean#L474-L475